pagerduty/user_test.go

```diff
@@ -259,10 +261,7 @@ func TestListAllWithLicenses(t *testing.T) {
 			License: &LicenseReference{ID: "P1D3XYZ", Type: "license_reference"},
 		},
 	}
-
-	if !reflect.DeepEqual(resp, want) {
-		t.Errorf("returned %#v; want %#v", resp, want)
-	}
+	assert.Equal(t, want, resp)
```

Comment:
Yes, `testify` could help better with the output of the tests, however in this case that's not the source of the issue. Therefore, please revert this change I just submit an issue to address this bug #132